_build/templates/default/sass/index.scss

```diff
@@ -551,26 +559,28 @@ textarea.x-form-field {
   color: #999 !important;
 }
 
-a.x-grid-link {
-  color: $colorSplash;
-  text-decoration: underline;
-}
-
-a.x-grid-link:hover,
-a.x-grid-link:focus {
-  text-decoration: none;
+.x-grid-link {
+    @include textLink;
+    &.simulated-link {
+        cursor: pointer;
+    }
 }
 
 .x-editable-column {
-  cursor: pointer;
-  &:hover,
-  &:focus {
-    color: $colorSplash;
-    > div::after {
-      @include awesome-font;
-      content: fa-content($fa-var-pen);
-      margin: 0 0.5em;
+  cursor: default;
+  &:not(.editor-disabled) {
+    cursor: pointer;
+    &:hover,
+    &:focus {
       color: $colorSplash;
+      > div {
+        &::after {
+            @include awesome-font;
+            content: fa-content($fa-var-pen);
+            margin: 0 0.5em;
```

Comment:
nit

I prefer to use `rem` instead of `em`, because `rem` is independent of the DOM-hierarchy

Reply:
Yes, I'm with you on this one, but it's something to tackle across all the sass files at some point in a different PR.